File: Python/lib/Led.py
import RPi.GPIO as GPIO
import lib/LedStatus

class Person:
    def __init__(pin, status=LedStatus.OFF):
        self.pin = pin
        self.status = status
        self.setup()
        if self.status == LedStatus.OFF
            self.on()
        else
            self.off()

    def setup(self):
        GPIO.setup(self.pin, GPIO.OUT)

    def on(self):
        self.status
        GPIO.output(self.pin, GPIO.LOW)

    def off(self):
        self.status
        GPIO.output(self.pin, GPIO.LOW)

File: lib/Led.js
var Gpio = require('onoff').Gpio;

module.exports = class Led {
    constructor(pin) {
      this.LED = new Gpio(pin, 'out')
      this.LED.writeSync(0)
    }

    on() {
      this.LED.writeSync(0)
    }

    off() {
      this.LED.writeSync(1)
    }

    set(status=true) {
      if(status) {
        this.on()
      } else {
        this.off()
      }
    }

    status() {
        return this.LED.readSync() === 0
    }

    destroy() {
      this.off()
      this.LED.unexport()
    }
}
